### PR TITLE
Pets controller

### PR DESCRIPTION
**Pull request by polanpio**
Added home, index and show actions to pets controller plus all the corresponding views (no styles of course)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
   devise_for :users
-  root to: 'pages#home'
+  root to: 'pets#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :pets do
+    resources :bookings, only: [:create]
+  end
+  get "dashbord", to: 'dashboards#display'
```
Comment (ellie-xyb) [inline config/routes.rb]:
```suggestion
  get "dashboard", to: 'dashboards#display'
```

**Review by ellie-xyb — CHANGES_REQUESTED** [review]
截屏2021-07-18 17 26 24

Do you know where is this coming from? I couldn't figure it out.

Also, the commit mention views, I didn't find the files. Should they be here?

File: config/routes.rb (hunk 1 above)
Comment (polanpio) [reply config/routes.rb]: good catch! I will change it!

Comment (ellie-xyb) [reply config/routes.rb]: Just commit the suggestion in one branch only, or it will conflict.

**Pull request merged**